client/src/stores/entryPointStore.js

```diff
@@ -0,0 +1,42 @@
+import { defineStore } from "pinia";
+import { getAppRoot } from "onload/loadConfig";
+import { rethrowSimple } from "utils/simple-error";
+import axios from "axios";
+
+export const useEntryPointStore = defineStore("entryPointStore", {
+    state: () => ({
+        entryPoints: [],
+        interval: null,
```

Comment:
this is probably not a good idea? @dannon

Reply:
Storing `interval` as state?  I think that's fine -- we'll need to make sure that multiple components calling `startPolling` doesn't preempt or abandon existing requests, so below you could just check if there was an existing interval.